components/homme/src/preqx_kokkos/cxx/cxx_f90_interface_preqx.cpp

```diff
@@ -63,10 +63,6 @@ void init_simulation_params_c (const int& remap_alg, const int& limiter_option,
 
   if (remap_alg==1) {
     params.remap_alg = RemapAlg::PPM_MIRRORED;
-  } else if (remap_alg == 2) {
```

Comment:
does this supposed to have alg=10?

Reply:
This is for just preqx, so it doesn't matter since we're not adding new features to preqx.